modules/nf-core/checkm/qa/tests/main.nf.test

```diff
@@ -0,0 +1,49 @@
+nextflow_process {
+
+    name "Test Process CHECKM_QA"
+    script "../main.nf"
+    process "CHECKM_QA"
+    tag "modules"
+    tag "modules_nfcore"
+    tag "checkm"
+    tag "checkm/qa"
+    tag "checkm/lineagewf"
+
+    test("Test CheckM QA") {
+
+        setup {
+            run("CHECKM_LINEAGEWF") {
+                script "../../lineagewf/main.nf"
+                process {
+                    """
+                    input[0] = [
+                        [ id:'test', single_end:false ], // meta map
+                        file(params.modules_testdata_base_path + 'genomics/sarscov2/illumina/fasta/contigs.fasta', checkIfExists: true)
+                    ]
+                    input[1] = 'fasta'
+                    input[2] = [] // Download CheckM database
+                    """
+                }
+            }
+        }
+
+        when {
+            process {
+                """
+                input[0] = CHECKM_LINEAGEWF.out.checkm_output.join(CHECKM_LINEAGEWF.out.marker_file)
+                             .map { v -> v + [file('NO_FILE')] }
+                input[1] = []
+                """
+            }
+        }
+
+        then {
+            assertAll(
+                { assert process.success },
+                { assert snapshot(process.out).match() }
+            )
+        }
+
+    }
+
+}
```

Comment:
Can you add a test case for the stub section as well? Just copy paste the normal test and add `options "-stub"` like here: https://github.com/nf-core/modules/blob/9deee8f0a2f91fa0a09246c23efdb921054e59d3/modules/nf-core/bedtools/bamtobed/tests/main.nf.test#L34